### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/features.md` and `docs/uk/docs/index.md`

### PR DESCRIPTION
**Pull request by shidenko97**
The pull request creates Ukrainian translation for online documentation of FastAPI. 
Can be part of the Pull Request #1790 (We can combine the best parts of translations to one perfect).
Relates to issue #1748

### CODE REVIEW

**Comment by tiangolo** [comment]
Thanks for your work! :globe_with_meridians: 

I just created a new PR with just the setup for the language, without the translations, so that you and others can continue to translate other pages while waiting for others to review the ones that are ready.

Maybe @ArcLightSlavik could review this?

**Comment by ArcLightSlavik** [comment]
This commit https://github.com/tiangolo/fastapi/pull/1791/commits/d8d6289585290187d836fc8f675c003b6accbf91 introduced conflicts, i assume docs preview on netlify does not work because of it

**Comment by shidenko97** [comment]
> This commit [d8d6289](https://github.com/tiangolo/fastapi/commit/d8d6289585290187d836fc8f675c003b6accbf91) introduced conflicts, i assume docs preview on netlify does not work because of it

Oh, yes
I will try to fix it soon as possible, Thanks

**Comment by shidenko97** [comment]
@ArcLightSlavik I've fixed the merge conflict, now documentation is shown correctly

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:37]: завдяки Starlette та Pydantic

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:39]: Збільшує швидкість розробки на 200 - 300 відсотків.

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:40]: Зменшує можливість виникнення людських помилок на 40 відсотків.

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:41]: Хороша робота з редактором.

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:93]: Якщо ви розробляєте? (not sure)

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:118]: Don't think it's bad if we say production specifically instead of 'real project' even if the latter sounds slightly better

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:157]: Якщо Ваш код містить (`async` / `await`), використовуйте `async def`:
# Closer aligned to english version

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:208]: перезавантажує сервер після кожної зміни в коді. Використовується лише при розробці.

File: docs/uk/docs/index.md
Comment (ArcLightSlavik) [inline docs/uk/docs/index.md:378]: again not sure about upper case Вам/Ви

**Review by ArcLightSlavik — COMMENTED** [review]
I believe features should translation should be a separate pull request (every file separate pull request)
And still need to translate 'Opinions' page, both chinese and spanish versions do translate them.

File: docs/uk/docs/index.md
Comment (mvoitko) [inline docs/uk/docs/index.md:93]: CLI app перекладається разом
застосунок Інтерфейсу Командного Рядка

File: docs/uk/docs/index.md
Comment (mvoitko) [inline docs/uk/docs/index.md:118]: production це точно не реальний проект. Можливо "промислове середовище" або лишити "production" середовище

File: docs/uk/docs/index.md
Comment (mvoitko) [inline docs/uk/docs/index.md:224]: не треба перекладати __paths__ бо це назва атрибуту в коді

File: docs/uk/docs/index.md
Comment (LikoIlya) [reply docs/uk/docs/index.md:118]:
```suggestion
Також Вам потрібен ASGI сервер для production етапу, наприклад <a href="http://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> або <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [reply docs/uk/docs/index.md:208]:
```suggestion
* `--reload`: перезавантажує сервер після кожної зміни в коді. Використовується лише при розробці.
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:95]:
```suggestion
**Typer** це молодший брат FastAPI. Він призначенний бути **FastAPI для CLIs**. ⌨️ 🚀
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:157]:
```suggestion
Якщо Ваш код містить `async` / `await`, використовуйте `async def`:
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:185]:
```suggestion
Запустіть сервер за допомого команди:
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:224]:
```suggestion
* Отримує HTTP запит на _paths_ `/` та `/items/{item_id}`.
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:225]:
```suggestion
* Обидва _paths_ отримують `GET` <em>операції</em> (також відомі як HTTP _methods_).
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:227]:
```suggestion
* _Path_ `/items/{item_id}` має _path parameter_ `item_id`, що має бути типом `int`.
* _Path_ `/items/{item_id}` має необов'язковий _query parameter_`q`, типу `str`.
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:233]:
```suggestion
Ви побачити автоматично-генеровану та інтерактивну API документацію (надану за допомогою <a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>):
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:241]:
```suggestion
Тут Ви можете побачити альтернативну автоматично-генеровану документацію (надану за допомогою <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
```

File: docs/uk/docs/index.md
Comment (LikoIlya) [inline docs/uk/docs/index.md:291]:
```suggestion
* Натисніть на кнопку "Try it out" - це дозволить Вам заповнити параметри та взаємодіяти з API напряму:
```

**Review by LikoIlya — APPROVED** [review]

File: docs/uk/docs/index.md
Comment (vald-phoenix) [reply docs/uk/docs/index.md:378]: I absolutely agree about the upper case. It more looks like a posh language with a reference to a "big cheese" though you can refer to one person (singular) with "Ви" it is completely wrong if you mean the plural.

File: docs/uk/docs/features.md
Comment (vald-phoenix) [reply docs/uk/docs/features.md:18]: Можна просто "тестуйте API напряму"

File: docs/uk/docs/features.md
Comment (ss-o-furda) [inline docs/uk/docs/features.md:201]: "оголошувати"

File: docs/uk/docs/features.md
Comment (ss-o-furda) [inline docs/uk/docs/features.md:198]: "легкими в оголошенні", "перевірЕними"

File: docs/uk/docs/features.md
Comment (ss-o-furda) [inline docs/uk/docs/features.md:189]: не впевнений що доречно використовувати тут

**Comment by tiangolo** [comment]
It would probably be better to split this into two PRs, one for each file. It would make it easier to review. Also, once one file is ready, others can approve the PR for that file, even if the other file is not ready yet.

Also, as @shidenko97 is probably busy and hasn't been able to continue with this, maybe others can continue the work in independent PRs. ☕

**Review by romashevchenko — APPROVED** [review]

File: docs/uk/docs/features.md
Comment (romashevchenko) [inline docs/uk/docs/features.md:179]:
```suggestion
**FastAPI** повністю сумісний з (і базується на) <a href="https://pydantic-docs.helpmanual.io" class="external-link" target="_blank"><strong>Pydantic</strong></a>. Тому, будь-який додатковий код Pydantic, який у вас є, також буде працювати.

```

File: docs/uk/docs/features.md
Comment (romashevchenko) [inline docs/uk/docs/features.md:193]:
```suggestion
    * Тому що pydantic структури даних всього-на-всього екземпляри класів, які Ви визначаєте; автодоповнення, linting, mypy та Ваша інтуїція мають працювати коректно з Вашою валідацією даних.
```

File: docs/uk/docs/features.md
Comment (romashevchenko) [inline docs/uk/docs/features.md:198]:
```suggestion
    * Також валідатори дозволяють чітко та легко визначати, перевіряти та документувати складні схеми даних як схему JSON. JSON Schema.
```

File: docs/uk/docs/features.md
Comment (romashevchenko) [inline docs/uk/docs/features.md:199]:
```suggestion
    * Ви можете мати глибокі **вкладені JSON** об'єкти та все це буде валідоване та анотоване.
```

File: docs/uk/docs/features.md
Comment (romashevchenko) [inline docs/uk/docs/features.md:201]:
```suggestion
    * Pydantic дозволяє визначати кастомні типи даних або Ви можете розширювати валідацію з методами декорованої моделі завдяки декоратору валідатора.
```

File: docs/uk/docs/index.md
Comment (romashevchenko) [inline docs/uk/docs/index.md:363]:
```suggestion
    * Так як `q` параметр оголошено з `= None` - він є необов'язковим.
```

File: docs/uk/docs/index.md
Comment (romashevchenko) [inline docs/uk/docs/index.md:366]:
```suggestion
    * Перевіряти, що міститься обов'язковий параметр `name` , який має бути типу `str`. 
```

File: docs/uk/docs/index.md
Comment (romashevchenko) [inline docs/uk/docs/index.md:367]:
```suggestion
    * Перевіряти, що міститься обов'язковий параметр `price`, який має бути типу `float`.
```

File: docs/uk/docs/index.md
Comment (romashevchenko) [inline docs/uk/docs/index.md:368]:
```suggestion
    * Перевіряти, що міститься необов'язковий параметр `is_offer`, який має бути типу `bool`, якщо наявний.
```

File: docs/uk/docs/index.md
Comment (romashevchenko) [inline docs/uk/docs/index.md:398]:
```suggestion
...і побачите як Ваш редактор автоматично підставить атрибути та їх типи:
```

File: docs/uk/docs/index.md
Comment (romashevchenko) [inline docs/uk/docs/index.md:421]:
```suggestion
Незалежні TechEmpower бенчмарки показують, що **FastAPI** додатки, запущені з Uvicorn <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">є одними з найшвидших Python фреймворків на сьогоднішній день</a>, лише запущені з  Starlette та Uvicorn (використовуються всередині FastAPI). (*)
```

**Review by romashevchenko — COMMENTED** [review]
Hi!
I'm Romana from DreamProIT. I'm a native speaker, I made a few edits and suggestions to improve this translation.

**Review by romashevchenko — COMMENTED** [review]
Hi!
I'm Romana from DreamProIT. I'm a native speaker, I made a few edits and suggestions to improve this translation.

**Review by romashevchenko — COMMENTED** [review]
Hi!
I'm Romana from DreamProIT. I'm a native speaker, I made a few edits and suggestions to improve this translation.

**Comment by tiangolo** [comment]
As this has been quiet for a long time, I'm gonna close this one, if anyone wants to tackle any of these files you can create a new PR, feel free to copy things from this one or start from scratch. ☕

**Pull request closed without merging**
